control/autoware_vehicle_cmd_gate/src/vehicle_cmd_gate.cpp

```diff
@@ -457,10 +459,21 @@ void VehicleCmdGate::onTimer()
     }
   }
 
+  const auto CheckTopicContinuity = []<class T>(const T & prev_topic, const T & current_topic) {
+    return (rclcpp::Time(current_topic.stamp) - rclcpp::Time(prev_topic.stamp)).seconds() > 0.0
+             ? current_topic
+             : prev_topic;
+  };
+
+  prev_commands_.turn_indicator =
+    CheckTopicContinuity(prev_commands_.turn_indicator, turn_indicator);
+  prev_commands_.hazard_light = CheckTopicContinuity(prev_commands_.hazard_light, hazard_light);
+  prev_commands_.gear = CheckTopicContinuity(prev_commands_.gear, gear);
```

Comment:
Better to start the function name in lowercase to align other functions in this file. And this function does not just check, but return the appropriate topic, thus it is better to change the name. The below is an example.

```suggestion
  const auto getContinuousTopic = []<class T>(const T & prev_topic, const T & current_topic) {
    return (rclcpp::Time(current_topic.stamp) - rclcpp::Time(prev_topic.stamp)).seconds() > 0.0
             ? current_topic
             : prev_topic;
  };

  prev_commands_.turn_indicator =
    getContinuousTopic(prev_commands_.turn_indicator, turn_indicator);
  prev_commands_.hazard_light = getContinuousTopic(prev_commands_.hazard_light, hazard_light);
  prev_commands_.gear = getContinuousTopic(prev_commands_.gear, gear);
```